featuretools/variable_types/variable.py

```diff
@@ -121,97 +125,196 @@ def interesting_values(self, values):
 
 
 class Boolean(Variable):
-    """Represents variables that take on one of two values"""
-    _dtype_repr = "boolean"
+    """Represents variables that take on one of two values
+
+    Args:
+        true_values (list) : List of valued true values. Defaults to [1, True, "true", "True", "yes", "t", "T"]
+        false_values (list): List of valued false values. Defaults to [0, False, "false", "False", "no", "f", "F"]
+    """
+    type_string = "boolean"
     _default_pandas_dtype = bool
 
+    def __init__(self,
+                 id,
+                 entity,
+                 name=None,
+                 true_values=None,
+                 false_values=None):
+        default = [1, True, "true", "True", "yes", "t", "T"]
+        self.true_values = true_values or default
+        default = [0, False, "false", "False", "no", "f", "F"]
+        self.false_values = false_values or default
+        super(Boolean, self).__init__(id, entity, name=name)
+
+    def create_data_description(self):
+        description = super(Boolean, self).create_data_description()
+        description['properties'].update({
```

Comment:
this is should to go into the `type` field not `properties`

Reply:
Change applied in [this commit.](https://github.com/Featuretools/featuretools/pull/361/commits/bbfb55bf48a0d154860e80a4c53ea92873b4cc28)